src/webui/api/searchcontroller.cpp

```diff
@@ -300,6 +314,8 @@ void SearchController::checkForUpdatesFinished(const QHash<QString, PluginVersio
         LogMsg(tr("Updating plugin %1").arg(pluginName), Log::INFO);
         pluginManager->updatePlugin(pluginName);
     }
+
+    setResult(u""_s);
```

Comment:
This addition results in two changes to maintain backwards compatibility:
- API responds with 200 status code
- Response contains `Content-Type: text/plain; charset=UTF-8` header, as before


Without this line, the API would respond with a 204 and omit the `Content-Type` header (due to a lack of content).